frontend/packages/citizen-frontend/src/applications/editor/ApplicationFormData.ts

```diff
@@ -1,11 +1,14 @@
 import LocalDate from '@evaka/lib-common/src/local-date'
 import {
+  Attachment,
   Application,
   ApplicationForm,
   ApplicationPersonBasics
 } from '~applications/types'
+import { UUID } from '~../../lib-common/src/types'
 
 export type ServiceNeedFormData = {
+  applicationId: UUID
```

Comment:
I don't think applicationId should be part of form data. Rather pass it in props or read it from router.